app/styles/fonts/openshift-logos-icon/README.md

```diff
@@ -0,0 +1,10 @@
+OpenShift Icon Font Logos
+===========================
+
+To view a list of all the glyphs in the font along with their codes/ligatures, follow the [Getting Started](https://github.com/openshift/origin-web-console#getting-started) instructions to launch the console running locally and visit:
```

Comment:
so instead of this you should link them to https://rawgit.com/openshift/origin-web-console/master/app/styles/fonts/openshift-logos-icon/demo.html

which will exist once the PR is merged to master

you can see an example based on your branch here https://rawgit.com/sg00dwin/origin-web-console/openshift-logo-icon-updates/app/styles/fonts/openshift-logos-icon/demo.html